### PR TITLE
A4 julius

### PR DESCRIPTION
**Pull request by julwil**

### CODE REVIEW

**Review by jvdassen — APPROVED** [review]
Nice clean code IMO, good naming.

**Review by jvdassen — APPROVED** [review]
Interesting, doing the benchmarking with pure Java. Looks good

**Comment by jvdassen** [comment]
It all looks fine, just two things:
- I think the build is failing due to the `DocumentLoadTest` test. Maybe it would be a good idea to add an assertion message?
- The commit messages mention "a3-..." I assume this is just a typo right?

**Review by RinorSefa — COMMENTED** [review]
The code looks clean, I'm bit unclear of the information worth that we get from pure java performance testing and its reliability, but it makes sense in paper doing this way, I do not know what might happen in the background.

File: test/src/edu/stanford/nlp/simple/DocumentLoadTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    this.baseLine = end - start;
+  }
+
+  @Test
+  public void testSentencesLessThanLinear() {
+    String input = this.sentence;
+    long epsilon = 1000;
+
+    for (int i = 1; i <= 100; i++) {
```
Comment (RinorSefa) [inline test/src/edu/stanford/nlp/simple/DocumentLoadTest.java]: I believe that loops get optimized, so they not suggested.

**Comment by julwil** [comment]
@jvdassen @RinorSefa 
Thanks guys for the feedback. I removed the LoadTest as it failed on the remote machine. As Rinor already pointed out, it served more an 'academic' purpose rather than production code 👍

**Review by RinorSefa — APPROVED** [review]

**Pull request merged**